graylog2-web-interface/src/components/grok-patterns/GrokPatterns.jsx

```diff
@@ -153,21 +204,36 @@ const GrokPatterns = createReactClass({
         <Row className="content">
           <Col md={12}>
             <IfPermitted permissions="inputs:read">
-              <GrokPatternsList id="grok-pattern-list"
-                                className="table-striped table-hover"
-                                headers={headers}
-                                headerCellFormatter={this._headerCellFormatter}
-                                sortByKey="name"
-                                rows={this.state.patterns}
-                                dataRowFormatter={this._patternFormatter}
-                                filterLabel="Filter patterns"
-                                filterKeys={filterKeys} />
+              <Row className="row-sm">
+                <Col md={8}>
+                  <SearchForm onSearch={this._onSearch}
+                              onReset={this._onReset}
+                              queryHelpComponent={queryHelperComponent}
+                              useLoadingState />
+                </Col>
+              </Row>
+              <Row>
+                <Col md={12}>
+                  <PaginatedList onChange={this._onPageChange}
+                                 totalItems={pagination.total}>
+                    <br />
+                    <br />
```

Comment:
Can we use styles here instead of HTML?

Reply:
I do not think it is needed at all. But I would like your input here.

Reply:
I would rather them be deleted if you think it still renders fine. 
If you feel like the spacing would help the UI, then it would be best if it is implemented as margin or padding in a styled-component.